platform-ajax/src/main/typescript/com/softicar/platform/ajax/engine/ForwardDeclarations.ts

```diff
@@ -0,0 +1,7 @@
+
+declare let SESSION_TIMED_OUT: boolean;
+
+declare function SR_sendAjaxRequest(message: any, form: any): void;
```

Comment:
That `SR_` seems cryptic. When we're already at it: Can we either remove the prefix, or spell it out?

Reply:
not in this PR, as you can see, it is only a forward declaration of a "module" I did not want to touch, yet.